sources/packages/web/src/assets/css/base.scss

```diff
@@ -362,6 +362,10 @@ strong {
   border: 1px solid $common-border-color;
 }
 
+.v-card.overflow {
```

Comment:
Apparently, we already have a solution for this for the `student-page-container`, no need to create a new style and it can be applied only to the container where it is needed.
```vue
 <student-page-container class="overflow-visible">
```

Reply:
Very nice solution. Thanks. 👍